apps/sports-helsinki/src/domain/search/combinedSearch/search.module.scss

```diff
@@ -98,26 +98,6 @@ $buttonWidth: 180px;
   }
 }
 
-.autoSuggestRowWithTargetGroupFilter {
-  @extend .autoSuggestRowBase;
-
-  & > :first-child {
-    @include respond-above(s) {
-      grid-column: 1 / 3;
-    }
-  }
-}
-
-.autoSuggestRowWithoutTargetGroupFilter {
-  @extend .autoSuggestRowBase;
-
-  & > :first-child {
-    @include respond-above(m) {
-      grid-column: 1 / 4;
-    }
-  }
-}
-
```

Comment:
This leaves the NEXT_PUBLIC_SHOW_TARGET_GROUP_FILTER feature flag functionality in a broken state i.e. you can't really turn off the target group filter anymore without breaking the user interface. But as the target group filter feature flag probably isn't needed anymore you could just remove it altogether IMHO.

Reply:
I did test that feature flag and it should work. Did you test it? I changed the CSS a bit.

Reply:
Seems to work ok in the less wide views but in the wider views it leaves an empty gap in the UI:

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/e40b4b8f-c2f4-4f5d-a96e-3fd147ccbaa2)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/a92b07be-95f5-4237-b4cf-bcd60bb2b902)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/a1e55d97-a50b-45ab-8ab3-ccbb170ffb0a)

![image](https://github.com/City-of-Helsinki/events-helsinki-monorepo/assets/77663720/ec9667b6-b9e7-4ac6-a357-1129873276bd)

Reply:
I think that I'll leave it as it is. There is that 1 column white space, but it should not matter since the taget group becomes always active field and the feature flag will be removed.